compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangNodeAnalyzer.java

```diff
@@ -619,7 +619,7 @@
 
     public abstract void visit(BLangWhile node, T data);
 
-    public abstract void visit(BLangWorkerSend node, T data);
+    public abstract void visit(BLangWorkerAsyncSendExpr node, T data);
```

Comment:
Lets first deprecate these and later remove it

Reply:
These are blang nodes, if we create a new expression for `AsyncSend` and direct it to a that blang node then these nodes can't be accessed at all. Hence no need to keep those nodes